pkg/monitortests/node/legacynodemonitortests/exclusions.go

```diff
@@ -0,0 +1,47 @@
+package legacynodemonitortests
+
+import (
+	"regexp"
+	"strings"
+
+	"github.com/openshift/origin/pkg/monitortestlibrary/platformidentification"
+)
+
+type Exclusion struct {
+	upgradeJob  bool
+	clusterData platformidentification.ClusterData
+}
+
+func isThisContainerRestartExcluded(locator string, exclusion Exclusion) bool {
+	// This is a list of known container restart failures
+	// Our goal is to conquer these restarts.
+	// So we are sadly putting these as exceptions.
+	// If you discover a container restarting more than 3 times, it is a bug and you should investigate it.
+	exceptions := []string{
+		"container/metal3-static-ip-set",      // https://issues.redhat.com/browse/OCPBUGS-39314
+		"container/ingress-operator",          // https://issues.redhat.com/browse/OCPBUGS-39315
```

Comment:
this only happens on single node clusters, so we should be sure that this exclusion is only valid for single node clsuters, not for HA clusters.